custom/icds_reports/utils/fix_child_health_missing_month.sql

```diff
@@ -0,0 +1,98 @@
+--  Custom Scan (Citus Router)  (cost=0.00..0.00 rows=0 width=0)
+--    Task Count: 64
+--    Tasks Shown: One of 64
+--    ->  Task
+--          Node: host=100.71.184.232 port=6432 dbname=icds_ucr
+--          ->  Update on child_health_monthly_323196 child_health  (cost=1.11..5.12 rows=1 width=661)
+--                Update on child_health_monthly_default_102648 child_health_1
+--                ->  Nested Loop  (cost=1.11..5.12 rows=1 width=661)
+--                      ->  Index Scan using icds_dashboard_growth_mo_month_state_id_9dfbeda1_idx_102264 on icds_dashboard_growth_monitoring_forms_102264 gm  (cost=0.56..2.25 rows=1 width=100)
+--                            Index Cond: (month = '2018-05-01'::date)
+--                      ->  Index Scan using child_health_monthly_default_102648_pkey on child_health_monthly_default_102648 child_health_1  (cost=0.56..2.78 rows=1 width=525)
+--                            Index Cond: ((supervisor_id = gm.supervisor_id) AND (case_id = (gm.case_id)::text) AND (month = '2018-05-01'::date))
+-- Second query to update agg_child_table
+
+UPDATE agg_child_health 
+ SET wasting_moderate = tmp.wasting_moderate,
+ 		 wasting_severe = tmp.wasting_severe,
+ 		 wasting_normal = tmp.wasting_normal,
+ 		 wasting_normal_v2 = tmp.wasting_normal_v2,
+ 		 wasting_moderate_v2 = tmp.wasting_moderate_v2,
+ 		 wasting_severe_v2 = tmp.wasting_severe_v2,
+ 		 stunting_moderate = tmp.stunting_moderate,
+ 		 stunting_severe = tmp.stunting_severe,
+ 		 stunting_normal = tmp.stunting_normal
+ 	FROM (
+ 		SELECT
+ 			chm.supervisor_id as supervisor_id,
+ 			chm.awc_id as awc_id,
+ 			chm.month as month,
+ 			SUM(CASE WHEN chm.current_month_wasting = 'moderate' THEN 1 ELSE 0 END) as wasting_moderate,
+ 			SUM(CASE WHEN chm.current_month_wasting = 'severe' THEN 1 ELSE 0 END) as wasting_severe,
+ 			SUM(CASE WHEN chm.current_month_stunting = 'moderate' THEN 1 ELSE 0 END) as stunting_moderate,
+ 			SUM(CASE WHEN chm.current_month_stunting = 'severe' THEN 1 ELSE 0 END) as stunting_severe,
+ 			SUM(CASE WHEN chm.current_month_wasting = 'normal' THEN 1 ELSE 0 END) as wasting_normal,
+ 			SUM(CASE WHEN chm.current_month_stunting = 'normal' THEN 1 ELSE 0 END) as stunting_normal,
+ 			SUM(CASE WHEN chm.zscore_grading_wfh_recorded_in_month = 1 AND chm.zscore_grading_wfh = 3 THEN 1 ELSE 0 END) as wasting_normal_v2,
+ 			SUM(CASE WHEN chm.zscore_grading_wfh_recorded_in_month = 1 AND chm.zscore_grading_wfh = 2 THEN 1 ELSE 0 END) as wasting_moderate_v2,
+ 			SUM(CASE WHEN chm.zscore_grading_wfh_recorded_in_month = 1 AND chm.zscore_grading_wfh = 1 THEN 1 ELSE 0 END) as wasting_severe_v2
+ 		FROM child_health_monthly chm
+      WHERE chm.month = '2018-05-01'
+            AND chm.supervisor_id >= '0'
+            AND chm.supervisor_id < '1'
+      GROUP BY chm.supervisor_id, chm.awc_id,
+               chm.month, chm.sex, chm.age_tranche, chm.caste 
+ 	) as tmp
+ 	WHERE supervisor_id = tmp.supervisor_id
+ 		AND awc_id = tmp.awc_id
```

Comment:
This wont be just awc_id=awc_id right? because in the inner query it is not only grouped by awc_id. its also grouped by sex, age_trance etc. could you please make the mapping correct.
This agg_child_health table is also grouped by other columns along with location columns
`    where (
        agg_child_health.awc_id=ut.awc_id and 
        agg_child_health.month=ut.month and
        agg_child_health.gender=ut.sex and
        agg_child_health.age_tranche=ut.age_tranche and
        agg_child_health.caste=ut.caste and
        agg_child_health.disabled=ut.coalesce_disabled and
        agg_child_health.minority = ut.coalesce_minority and
        agg_child_health.resident = ut.coalesce_resident`


Also I doubt it will work like this because child_health_monthly is a distritbuted table and agg_child_health is not.